go.mod

```diff
@@ -5,7 +5,7 @@ go 1.14
 require (
 	github.com/coreos/go-semver v0.3.0
 	github.com/fatih/color v1.9.0
-	github.com/ghodss/yaml v1.0.0
+	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
```

Comment:
Let's make sure this gets fixed before merging.

Reply:
I am not certain how to do that, its automatically done when i import ``valuemodifier` as it has it in `gomod - requires`
https://github.com/giantswarm/valuemodifier/blob/master/go.mod#L6

Reply:
I tried to get this straight in https://github.com/giantswarm/valuemodifier/pull/32.

Reply:
sorted